config/abilities/master_member.rb

```diff
@@ -17,13 +17,16 @@
     end
 
     can(:create, Account, &:signup_provider_possible?)
+
     if user.has_permission?(:partners)
       can :manage, :partners
+
+      can :manage, :provider_plans unless ThreeScale.config.onpremises
```

Comment:
Why don't we allow that on-premises?
I think we should

Reply:
@hallelujah Because it is what we do for master admin:
https://github.com/3scale/porta/blob/99ec8db6b7bd4b8286034cb212b4a1ecc6c5b016/config/abilities/master_admin.rb#L25-L38

Imo it is ok to allow it for on-prem, but then for both admin and members, not only for members 😝

Reply:
oh because we forbid creating more plans and more services etc ... on premises
I forgot that